modules/core/src/lib/layer.js

```diff
@@ -194,6 +199,10 @@ export default class Layer extends Component {
     return false;
   }
 
+  use64bitPositions() {
+    return this.props.fp64;
```

Comment:
Assume you are going to change this in next PR, otherwise should be same as above for now.

Reply:
Yes. I will change in next PR.